backend/src/common/services/services.ts

```diff
@@ -1,5 +1,10 @@
 import { CryptService } from './crypt/crypt.service.js';
+import { TokenService } from './token/token.services.js';
 
 const cryptService = new CryptService();
 
-export { cryptService };
+const secretKey = process.env['SECRET_KEY'] || 'default_secret_key';
```

Comment:
we have config class for handling .env

Reply:
And also https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

A1 rule here

Reply:
Done. Thank you